cmd/cstor-pool-mgmt/controller/new-pool-controller/handler.go

```diff
@@ -0,0 +1,244 @@
+/*
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package poolcontroller
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/golang/glog"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/controller/common"
+
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	apis2 "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha2"
+
+	pool "github.com/openebs/maya/pkg/cstor/pool/v1alpha2"
```

Comment:
lets import package name with zpool

Reply:
lets move this to cmd/cstor-pool-mgmt/pool/v1alpha2,
and similarly for wrappers around zfs command, lets move them to cmd/cstor-pool-mgmt/volumereplica/v1alpha2 and import it as zfs